Public/chess.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+	<link rel="stylesheet" type="text/css" href="chess_game.css">

```

Comment:
This linkage is wrong.  It should be <link rel="stylesheet" href="../assets/css/chess_game.css" />

Reply:
The link to the stylesheet needs to be changed because your chess_game.css file is stored in the assets folder

Reply:
Sure I will change it. Sorry for the inconveniences.